src/Fantomas/CodePrinter.fs

```diff
@@ -3215,6 +3215,10 @@ and genApp astContext e es ctx =
                 (fun ctx ->
                     match es with
                     | [] -> false
+                    | Paren _ :: rest when es.Length > 1 ->
```

Comment:
This is wrong, as soon as there are more than elements in `es` it should return true.
Unit test ```maintain indent if when condition is multiline``` should be updated as well.

Reply:
@nojaf 
It returns true if there are at least two curried args. It this fails, it continues with old logic.

Reply:
There is no need for the old logic, that is my point.